src/main/webapp/app/course/competencies/forms/course-competency-form.component.ts

```diff
@@ -1,4 +1,4 @@
-import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { Component, EventEmitter, Output, input } from '@angular/core';
```

Comment:
Is there a reason for changing the order here?

Reply:
Yes, input is now no longer capitalised (signal vs. `@Input()`) and our linter wants all capitalised words before the others